src/main/java/edu/usf/sas/pal/muser/io/CSVFileWriter.java

```diff
@@ -43,7 +43,7 @@ public void createHeader(String[] header) {
         csvWriter.writeNext(header);
     }
 
-    public void appendAllToCsV(List<MusicAnalysisModel> musicAnalysisModel) {
+    public void appendAllToCSV(List<MusicAnalysisModel> musicAnalysisModel) {
```

Comment:
Java conventions are to always use camel case even when something is typically capitalized, so this should be `appendAllToCsv()`